pom.xml

```diff
@@ -151,6 +152,15 @@
     <jakarta.validation-api.version>2.0.2</jakarta.validation-api.version>
     <javassist.version>3.30.2-GA</javassist.version>
     <jackson-jaxrs-json-provider.version>2.17.2</jackson-jaxrs-json-provider.version>
+    <!-- TODO Had added this to see if proprties are passed on via BOM.
```

Comment:
Marking as draft as this needs to revert.

Reply:
Yeah I don't think that properties are preserved via bom import. Instead, you'll need to add an explicit managed dependency for every dependency definition that's using one of the properties to manage a version.

Reply:
Sure I think we are in agreement here. Will revert the demo code.